features/container.feature

```diff
@@ -0,0 +1,6 @@
+Feature:
+
+  Scenario:
+    Given I have a Zend\ServiceManager container
```

Comment:
I like the explicitness, it's an example of a container.

Specifically this extension is for 'PSR-11 containers that are returned from an included file' which is not all of them (but applies for Zend)